setup/README.md

```diff
@@ -42,11 +42,19 @@ On your server:
 #### Adding more Wordpress Plugins
 Add the plugins to composer.json (search for the [here](https://wpackagist.org)); a few are in there already as examples.
 
-#### Connecting to the MySql Container from Sequel Pro?
+#### Connecting to the MySql Container from Sequel Pro.
```

Comment:
Is this specific to Sequel Pro or might there be other SQL tools that could work as well?